DEVELOP.md

```diff
@@ -85,10 +85,12 @@ will also detect packaging faults (e.g. omitted binaries & support files).
 
 Rigs tests are broken up into three groups which can be run as follows:
 
-    $ py.test tests  # The Rig test-suite
-    $ py.test rig    # Doctests in rig source code
-    $ py.test docs   # Doctests in Sphinx documentation whose
-                     # filename ends with '_doctest.rst'.
+    $ py.test tests                                # The Rig test-suite
+    $ py.test rig --doctest-modules                # Doctests in rig source code
+    $ py.test docs --doctest-glob='*_doctest.rst'  # Doctests in Sphinx
```

Comment:
This is just a bug in Github's markdown source rendering I think. Since it is in a literal code block it is [rendered correctly](https://github.com/project-rig/rig/blob/travis-py35/DEVELOP.md#running-tests) so doesn't need any escaping.